DecaDuino.h

```diff
@@ -94,8 +94,9 @@
 #define DecaDuino_h
 
 #include "Arduino.h"
+#include <math.h>
 
-//#define DECADUINO_DEBUG
+#define DECADUINO_DEBUG
```

Comment:
```suggestion
//#define DECADUINO_DEBUG
```
Debug should be disabled by default